src/api/configure.ts

```diff
@@ -54,6 +58,12 @@ export function configure(options: {
     if (computedRequiresReaction !== undefined) {
         globalState.computedRequiresReaction = !!computedRequiresReaction
     }
+    if (derivationRequiresObservable !== undefined) {
```

Comment:
Shouldn't both of these be enabled by default? I mean it just adds some safety checks which is a good thing. If someone wants to feel less protected, they can opt-out.

Reply:
I went with the spirit of `computedRequiresReaction`, that is not on by default. 
I personally think developers will benefit from it, but i believe in soft changes